### PR TITLE
shutdown Velas & Oasis

### PR DESCRIPTION
**Pull request by namgold**

### CODE REVIEW

File: src/pages/About/styleds.tsx [hunk 1]
```diff
@@ -227,7 +227,7 @@ export const Powered = styled.div`
   justify-content: center;
   gap: 52px;
   & > * {
-    width: calc(100% / 6 - 52px);
+    width: calc(100% / 5 - 52px);
```
Comment (nguyenhoaidanh) [inline src/pages/About/styleds.tsx:230]: can we update it to dynamic?

Comment (namgold) [reply src/pages/About/styleds.tsx:230]: wdym

Comment (nguyenhoaidanh) [reply src/pages/About/styleds.tsx:230]: 100% / xxx.length - 52px

Comment (namgold) [reply src/pages/About/styleds.tsx:230]: I afraid not. This number is number of columns, it has nothing to do with the list length. We have to set it manually to ensure the UI not breaking.

**Review by nguyenhoaidanh — APPROVED** [review]

**Review by nguyenhoaidanh — APPROVED** [review]

**Pull request merged**